message/validation/const.go

```diff
@@ -37,11 +37,18 @@ const (
 
 const (
 	partialSignatureSize           = 96
-	partialSignatureMsgSize        = partialSignatureSize + rootSize + operatorIDSize + validatorIndexSize
-	maxPartialSignatureMessages    = 1000
-	partialSigMsgTypeSize          = 8 // uint64
-	maxPartialSignatureMsgsSize    = partialSigMsgTypeSize + slotSize + maxPartialSignatureMessages*partialSignatureMsgSize
-	maxEncodedPartialSignatureSize = maxPartialSignatureMsgsSize + maxPartialSignatureMsgsSize/encodingOverheadDivisor + 4
+	maxSizePartialSignatureMessage = partialSignatureSize + rootSize + operatorIDSize + validatorIndexSize
+	/// Pre fork values TODO remove after Blossom
+	maxSizePartialSignatureMessagesOld = 1000
```

Comment:
IMO `PreFork` suffix would be better. It's might be confusing what oldness is meant